modules/op-installing-tkn-on-linux-using-rpm.adoc

```diff
@@ -5,10 +5,10 @@
 :_content-type: PROCEDURE
 [id="installing-tkn-on-linux-using-rpm"]
 
-= Installing {pipelines-title} CLI (tkn) on Linux using an RPM
+= Installing {pipelines-title} CLI on Linux using an RPM
```

Comment:
Maybe use "the" in this title to be consistent with other similar instances in this PR.
```suggestion
= Installing the {pipelines-title} CLI on Linux using an RPM
```

Reply:
If you choose to use "the" here, update other similar titles in this PR.